platforms/shared/configuration/setup-k8s-environment.yaml

```diff
@@ -30,7 +30,7 @@
       git_protocol: "{{ item.gitops.git_protocol | default('https') }}"
       git_url: "{{ item.gitops.git_url }}"
       git_key: "{{ item.gitops.private_key | default() }}"
-      flux_version: "0.41.2"
+      flux_version: "2.3.0"
```

Comment:
Did you make other changes for Flux upgrade?

Reply:
Yes, with Flux version 2.3.0, we need to update our `HelmRelease` to use the new `v2` API version instead of `v2beta1`.

Reply:
For the Indy platform, updated all Helm release templates to the latest version. 

However, for other platforms, the apiVersion is still set to `v2beta1`. Using `v2beta1` with the new version 2.3.0 will generate a warning to update the apiVersion to `v2`.

Reply:
Shall I create a separate PR (Quick Fix PR) for the Flux version upgrade? This will allow me to manage the files related to Indy and Flux separately and effectively.

Reply:
Yes, I will allow this to go ahead, and raise a separate PR for version upgrade